modules/ROOT/pages/update-acm.adoc

```diff
@@ -88,6 +89,17 @@ If you are upgrading from a package version before 1.74, follow these steps:
 
 Return to the main list of upgrade instructions and continue to follow them.
 
+[[pre-177]]
+== Pre-Install Steps When Upgrading from a Version before 1.77
+
+If you are upgrading from a package version before 1.77, follow these steps:
+
+. Open the *Developer Console* and open the *Query Editor*.
+. Enter `SELECT Id, Name, Owner.Name, Owner.IsActive FROM acm_pkg__CommunityApi__c WHERE Owner.IsActive = false` and click *Execute*.
+. If the query lists any users, change their status to Active with the https://help.salesforce.com/articleView?id=000336144&type=1&mode=1[reactivate a user] instructions.
```

Comment:
Font for Active OK?

Reply:
Good point. UI says "Active" but I think I'll just use "active" and make it part of the sentence.